apl-core/src/test/resources/logback-test.xml

```diff
@@ -53,6 +53,9 @@
             <logger name="com.apollocurrency.aplwallet" level="debug" additivity="false">
                 <appender-ref ref="STDOUT" />
             </logger>
+            <logger name="com.apollocurrency.aplwallet.apl.core.shard.helper.csv.CsvReaderTest" level="trace" additivity="false">
```

Comment:
Pls ,Comment out  to avoid a lot of unnecessary  logs during build

Reply:
This logger is optional, and disabled by default. 
This change doesn't affect the number of logs during the build.

Reply:
Look through test-env.properties